tests/test_http_graphql_query.py

```diff
@@ -3227,6 +3227,207 @@ def test_graphql_functional_order_04(self):
             ]
         })
 
+    def test_graphql_functional_exists_01(self):
+        self.assert_graphql_query_result(r"""
+            query {
+                User(
+                    filter: {profile: {exists: true}}
```

Comment:
Can you add a test where there's a link `exists`?

Reply:
There are link/property names that will fail to produce a GraphQL reflection, such as `and`, `or`, `not`. The name `exists` falls into the same category. We currently don't have specific tests for those either. There are probably weirder name clashes possible that cannot be resolved.

Should I add a bunch of test schemas for names that are legal in EdgeDB, but fail in our GraphQL reflection?

Reply:
Hm. Maybe *our* graphql "clauses" should be prefixed with `__`?  How does Hasura deal with this?

Reply:
Hasura only allows operators on fields, and since scalars have no properties there's no conflict.  Our issue is that we allow operating on objects and so operator names can clash with property names.

> Maybe our graphql "clauses" should be prefixed with `__`

This will inconvenience 99.9% of users for the benefit of 0.1%.  I think it's fine if we reflect properties that conflict with GraphQL "operator" keywords with a `_` suffix, e.g. `filter_` and couple that with good error messages and documentation.

Reply:
Maybe there's also way to use `@annotations` to disambiguate? (I don't quite recall where they can appear though)